app/packs/src/apps/mydb/elements/details/researchPlans/researchPlanTab/ResearchPlanDetailsFieldImage.js

```diff
@@ -60,38 +65,43 @@ export default class ResearchPlanDetailsFieldImage extends Component {
   }
 
   handleResizeChange(event) {
+    const zoom = event.target.value;
+    this.setState({ zoom });
     const { field, onChange } = this.props;
-    field.value.zoom = event.target.value;
+    field.value.zoom = zoom;
     onChange(field.value, field.id);
   }
 
   renderEdit() {
     const { field } = this.props;
     const currentAttachment = this.props.researchPlan.getAttachmentByIdentifier(field.value.public_name);
-    const is_annotationUpdated = currentAttachment != null && currentAttachment.updatedAnnotation;
+    const isAnnotationUpdated = currentAttachment != null && currentAttachment.updatedAnnotation;
+    const { zoom } = this.state;
     let content;
     if (field.value.public_name) {
-      const style = (field.value.zoom == null || typeof field.value.zoom === 'undefined'
-        || field.value.width === '') ? { width: 'unset' } : { width: `${field.value.zoom}%` };
+      const style = (zoom == null || typeof zoom === 'undefined'
+        || field.value.width === '') ? { width: 'unset' } : { width: `${zoom}%` };
       content = (
-        <div className="image-container">
-          <img style={style} src={this.state.imageSrc} alt={field.value.file_name} />
+        <div>
+          <img style={style} src={this.state.imageSrc} alt={field.value.file_name}
+            className="img-fluid"
+          />
         </div>
       );
     } else {
       content = <p>Drop Files, or Click to Select.</p>;
     }
     return (
       <div>
-        <FormGroup style={{ width: '30%' }}>
+        <Form.Group style={{ width: '30%' }}>
```

Comment:
Maybe you can try the col classes for the width
https://getbootstrap.com/docs/5.3/layout/grid/